packages/web-components/src/components/va-date/va-date.tsx

```diff
@@ -186,17 +169,20 @@ export class VaDate {
                   <option value={day}>{day}</option>
                 ))}
             </va-select>
-            <va-number-input
+            <va-text-input
               label="Year"
               name={`${name}Year`}
-              max={maxYear}
-              min={minYear}
+              maxlength={4}
+              minlength={4}
+              pattern="[0-9]{4}"
               // Value must be a string
-              value={year?.toString()}
+              // Checking is NaN if so provide empty string
+              value={year ? year.toString() : ''}
```

Comment:
This is minor but I think we could do `year?.toString() || ''`

Reply:
@andresriveratoro so I tried that originally but it is evaluating as truthy so we end up getting this:
<img width="705" alt="Screen Shot 2022-05-04 at 9 16 07 AM" src="https://user-images.githubusercontent.com/11822533/166689050-2c24a3e2-e5ef-4e38-bce0-36f90c6c9c92.png">
Which is why I switched it to a ternary.